swap/src/monero/wallet_rpc.rs

```diff
@@ -13,23 +12,28 @@ use tokio::{
     io::{AsyncBufReadExt, AsyncWriteExt, BufReader},
     process::{Child, Command},
 };
-use tokio_tar::Archive;
 use tokio_util::{
     codec::{BytesCodec, FramedRead},
     io::StreamReader,
 };
 
+#[cfg(not(any(target_os = "macos", target_os = "linux", target_os = "windows")))]
+compile_error!("unsupported operating system");
+
 #[cfg(target_os = "macos")]
 const DOWNLOAD_URL: &str = "http://downloads.getmonero.org/cli/monero-mac-x64-v0.17.1.9.tar.bz2";
 
 #[cfg(target_os = "linux")]
 const DOWNLOAD_URL: &str = "https://downloads.getmonero.org/cli/monero-linux-x64-v0.17.1.9.tar.bz2";
 
-#[cfg(not(any(target_os = "macos", target_os = "linux")))]
-compile_error!("unsupported operating system");
+#[cfg(target_os = "windows")]
+const DOWNLOAD_URL: &str = "https://downloads.getmonero.org/cli/monero-win-x64-v0.17.1.9.zip";
 
 const PACKED_FILE: &str = "monero-wallet-rpc";
```

Comment:
I think this might need a `cfg(not(target_os = "windows"))` otherwise you will be getting a "const already defined error".

Reply:
Yep, I updated, moved it up before the constant declarations